utils/sidekick.js

```diff
@@ -13,7 +21,7 @@ export async function messageSidekick(message, callback) {
     if (chrome && chrome.runtime && chrome.runtime.sendMessage) {
```

Comment:
this can probably be a different PR. but I sorta think if `window.chrome` isn't defined, we should still resolve the promise with `resolve(NO_SIDEKICK);` so that users in non chrome browsers still get some feedback when the click the profile icon.